### PR TITLE
Update teams-and-skypeforbusiness-coexistence-and-interoperability.md

### PR DESCRIPTION
**Pull request by luis-msftpm**
Updating some routing details as we rollout the changes to always route natively for Island users when the originator is on Teams.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/teams-and-skypeforbusiness-coexistence-and-interoperability.md [hunk 1]
```diff
@@ -67,7 +67,7 @@ To avoid a confusing or regressed Skype for Business experience, the Skype for B
 Phone System isn't supported in Teams in **Islands** mode. **Islands** mode doesn't support Enterprise Voice client is Skype for Business.
```
Comment (JohanFreelancer9) [inline Teams/teams-and-skypeforbusiness-coexistence-and-interoperability.md:67]:
```suggestion
Phone System isn't supported in Teams in **Islands** mode. **Islands** mode doesn't support Enterprise Voice client in Skype for Business.
```

File: Teams/teams-and-skypeforbusiness-coexistence-and-interoperability.md [hunk 2]
```diff
@@ -310,17 +310,13 @@ For example, Assume User A is in Islands mode. If User A is active in Teams but
...
+- If the receiver and sender are in TeamsOnly upgrade or Islands mode, the conversation will be a native chat experience which includes all the rich messaging and calling capabilities. To learn more, read [Native chat experience for external (federated) users in Teams](native-chat-for-external-users.md). 
 
-- If either of the conversation participants is NOT in TeamsOnly upgrade mode, the conversation remains an interop experience with text-only messages. The user interface exposes federated chats in a similar manner to same-tenant interop threads, except there is a note indicating the user is external.
+- If either of the conversation participants is on SfB* upgrade mode, the conversation remains an interop experience with text-only messages. The user interface exposes federated chats in a similar manner to same-tenant interop threads, except there is a note indicating the user is external.
```
Comment (JohanFreelancer9) [inline Teams/teams-and-skypeforbusiness-coexistence-and-interoperability.md:319]:
```suggestion
- If either of the conversation participants is in Skype for Business upgrade mode, the conversation remains an interop experience with text-only messages. The user interface exposes federated chats in a similar manner to same-tenant interop threads, except there is a note indicating the user is external.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @luis-msftpm, kindly commit the additional suggested changes. Many thanks.

**Pull request closed without merging**